managedtenants/utils/ocm.py

```diff
@@ -162,6 +165,36 @@ def disable_addon(self, addon_id):
             f"/api/clusters_mgmt/v1/addons/{addon_id}", json={"enabled": False}
         )
 
+    def shortcircuit_migrate(self, addon_id):
+        def addon_migration_complete():
+            migration = self.get_addon_migration(addon_id).json()
+            return migration.get("state") == "completed"
```

Comment:
```suggestion
        addon_migration_complete = lambda _: self.get_addon_migration(addon_id).json().get("state") == "completed"
```

imo we can use the benefits of lambda function here :)

Reply:
I thought of using it as well. But the linter is not happy about the line length and as far as i know there is no way to split a lambda function into multiple lines in python. :/

Reply:
ahh gotcha, we could skip the linter check but ehh too much hassle.